app/services/SubmissionService.scala

```diff
@@ -50,10 +50,13 @@ class SubmissionService @Inject()(
 
         auditService.auditAllPagesUserInput(choice, Json.toJson(cacheMap).as[JsObject])
 
+        val movementAuditType =
+          if (choice.value == Choice.AllowedChoiceValues.Arrival) AuditTypes.Arrival else AuditTypes.Departure
```

Comment:
Using the same naming will be confusing during import, I know that it's in different objects but in practice it always be confusing. My propose is to change it slightly to know what Arrival is for Choice and what for Audit